nucleus/dataset_item.py

```diff
@@ -40,28 +139,39 @@ def from_json(cls, payload: dict):
     def local_file_exists(self):
         return os.path.isfile(self.image_location)
 
-    def to_payload(self) -> dict:
-        payload = {
-            IMAGE_URL_KEY: self.image_location,
+    def to_payload(self, is_scene=False) -> dict:
+        payload: Dict[str, Any] = {
             METADATA_KEY: self.metadata or {},
         }
+
+        if is_scene:
+            if self.image_location:
+                payload[URL_KEY] = self.image_location
```

Comment:
Same comment as above: change the logic so serialization works properly regardless of whether this is a scene or not.

Reply:
same as above - serialization will work in both cases, but the payloads produced are different (since the `append` and `upload_scenes` endpoints expect diff formats

Reply:
Same as above: have only one JSON format for this class not two